### DISCUSSION TITLE
SQLite security

### DISCUSSION THREAD

**Discussion by subhadip1203** [pocketbase/pocketbase#3780] [Q&A]
As per the Project, the database is SQLite. 
Does Pocketbase use any encryption on the data saved on SQLite?
or anyone have any info on how to keep SQLite secure (on an Ubuntu server)

** I understand that application settings  (like OAuth2 clients, SMTP password, etc.) can be encrypted, but I am talking about whole database encryption.

**Comment by ganigeorgiev** [comment]
Similar to other databases, there is no encryption applied out of the box on the SQLite db file. You can also check https://github.com/pocketbase/pocketbase/discussions/1961.
